plugins/TelescopeControl/src/TelescopeControl.cpp

```diff
@@ -1551,7 +1551,7 @@ void TelescopeControl::loadDeviceModels()
 	//Clear the list of device models - it may not be empty.
 	deviceModels.clear();
 
-	//Cicle the list of telescope deifinitions
+	//Cycle the list of telescope deifinitions
```

Comment:
you missed "deifinitions" --> definitions.

Reply:
added in #4015